src/generators/outline.js

```diff
@@ -0,0 +1,7 @@
+import defineClasses from '../util/defineClasses'
+
+export default function() {
+  return defineClasses({
+    'no-outline': { outline: 'none' },
```

Comment:
Bootstrap uses `outline: 0` for theirs, not sure if it really matters but when there are multiple ways to do things I think it's a good idea to use precedent from a huge project with tons of users. Saves a couple bits in the compiled CSS too I guess.

Reply:
Yep, okay, will update.